CAIPs/caip-24.md

```diff
@@ -0,0 +1,69 @@
+---
+caip: 24
+title: Asset Reference for the Cosmos Asset Namespace
+author: Theobold Frazier
+discussions-to: https://github.com/shapeshift/CAIPs/issues/1
+status: Draft
+type: Standard
+created: 2020-02-14
+updated: 2020-02-14
+requires: 19
+---
+
+## Simple Summary
+
+This document is about the details of the Cosmos network asset namespace and reference for CAIP-19.
+
+## Abstract
+
+In CAIP-19 a general asset identification scheme is defined. This is the
+implementation of CAIP-19 for Cosmos chain native assets and ibc assets.
+
+## Motivation
+
+See CAIP-19.
+
+## Specification
+
+### Cosmos chain Native Asset Namespace
+
+An asset namespace called "native" that references Cosmos network blockchain native assets, allowing for unique identification on Cosmos chains with multiple native assets.
+
+### Cosmos chain ibc asset Asset Namespace
+
+The asset namespace is called "ibc". It reference Cosmos network blockchain inter blockchain channel assets.
```

Comment:
Should this standard work for single assets only or should it also handle pools format e.g `IBC/<SrcChainID>/<DstChainID>` ? If so, we should specify that the id might be `id` | `sourceChainId/distchainId`

Reply:
This standard was written to work for single assets. It appears this _namespace_ "ibc" would work for pool formats, but the asset _reference_ may be different unless there is a similar identifier produced for those pool assets.

I'm less familiar with cosmos pool assets, so I'm open to ideas.

Reply:
Yeah, precisely, this will already work for pool formats as is. I'm just wondering whether or not it would make sense to mention that the asset reference might be of pool format.
With that being said, in the context of this CAIP, that might be irrelevant: a reference is a reference, and consumers of this CAIP can then parse it however they need depending on its format, so we could keep this standard simpler by not mentioning it! More agnostic this way.